test/WebJobs.Script.Tests/Rpc/RpcInitializationServiceTests.cs

```diff
@@ -104,13 +105,44 @@ public async Task RpcInitializationService_Initializes_RpcServerAndChannels_Linu
             mockEnvironment.Setup(p => p.GetEnvironmentVariable(LanguageWorkerConstants.FunctionWorkerRuntimeSettingName)).Returns(string.Empty);
             mockEnvironment.Setup(p => p.GetEnvironmentVariable(EnvironmentSettingNames.ContainerName)).Returns("LinuxContainer");
 
+            var fileSystem = new Mock<IFileSystem>();
```

Comment:
move this to constructor

Reply:
Only a few tests need this. so method scoped might be enough.